docs/staging-instructions.md

```diff
@@ -0,0 +1,28 @@
+# Management of creating staging release
+
+The process of moving code from [Master](https://github.com/ITISFoundation/osparc-simcore/tree/master) branch to [Staging](https://github.com/ITISFoundation/osparc-simcore/tree/staging) branch is described here.
+
+## Process
+
+1. Create a PR from master to staging
+
```

Comment:
why not create a script for this instead of adding it as doc?

Reply:
you are right but I want people to RTFM instead of executing things without thinking. Especially if it then does not work out of the box for whatever reason. So let's start with the commands, then if that works well we can change them to a script.
Besides, the git commands are below and documented.